homepairs/homepairsUI/src/Screens/Main/ServiceRequest/ServiceRequestScreen/ServiceRequestScreen.tsx

```diff
@@ -39,10 +39,18 @@ const ServiceRequestScreen = connect(
     mapDispatchToProps,
 )(ServiceRequestScreenBase);
 
+const ServiceRequestScreenBaseWithNavigation = withNavigation(ServiceRequestScreen);
```

Comment:
We have moved away from using with Navigation. Please use prepareNavigationHandlerComponent instead. It is a helper function that helps give navigation to the component based on web or mobile platforms.